index.html

```diff
@@ -104,7 +102,7 @@ <h3 id="hook-three">a common place?</h3>
 
             <div class="about-button"> <!--Portfolio Button-->
```

Comment:
I recommend using `button` elements and styling them rather than creating buttons from scratch.

```html
<button class="about"><<< Portfolio</button>
```

Reply:
Word yeah, for some reason the styling I had wasn't applying correctly when I used the button element so I tried to make it from scratch. Perhaps I didn't target the element correctly. I think I'm still sort of fumbling with the concept of inherency/hierarchy when it comes to CSS styling.